### PR TITLE
Bug 1082688 - Translate the application name in AttentionWindow's notifi...

### PR DESCRIPTION
**Pull request by etiennesegonzac**
...cations r=alive

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=e0ab2510e805c8670a810d6aa4447895083084fe

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=593d1b2e25cc7e35706a5e88593cfc2cfbdd39c0

File: apps/system/js/attention_window.js [hunk 1]
```diff
@@ -220,10 +221,25 @@
 
     // Hide on creating.
     this.notification.style.display = 'none';
+
+    this.translateNotification();
+  };
+
+  AttentionWindow.prototype.translateNotification = function() {
+    if (this.notification) {
```
Comment (KevinGrandon) [inline apps/system/js/attention_window.js]: nit: just looking at ```AttentionWindow.prototype.ready/makeNotification```, it seems the general style for this file is an early return. I would opt to copy the pattern of the rest of the file to do an early return instead for consistency.

Comment (etiennesegonzac) [inline apps/system/js/attention_window.js]: ahah, I was probably the one asking for the other early returns ;)

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=eb58b28caff245d09df29cc8b9d05e21777a2925

**Pull request merged**